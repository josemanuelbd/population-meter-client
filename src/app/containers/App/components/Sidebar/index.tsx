import React from 'react';

const Sidebar: React.FC = () => <nav />;

export default Sidebar;
