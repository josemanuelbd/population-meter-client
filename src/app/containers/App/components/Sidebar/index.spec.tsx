import React from 'react';
import { shallow } from 'enzyme';

import Sidebar from '.';

describe('Sidebar', () => {
  it('should return a valid header', () => {
    const component = shallow(<Sidebar />);
    expect(component).toMatchSnapshot();
  });
});
