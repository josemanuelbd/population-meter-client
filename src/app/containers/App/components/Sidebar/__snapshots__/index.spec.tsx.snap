// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sidebar should return a valid header 1`] = `<nav />`;
